src/MainGrid.js

```diff
@@ -777,6 +799,20 @@ MainGrid.prototype.rangeToDomain = function(scale, value) {
     return scale.domain()[d3.bisect(scale.range(), value) - 1];
 };
 
+
+MainGrid.prototype.nullableObsLookup = function(donor, gene) {
+    var _self = this;
+
+    if (donor === undefined || donor === null) return null;
+    if (gene === undefined || donor === null) return null;
```

Comment:
this could probably be simplified to if(!donor || !gene) return null; seeing as all falsy values should all return null.
or

if(!donor || typeof donor !== 'object') return null;
if(!gene || typeof gene !== 'object') return null;